types/selrond__knit/selrond__knit-tests.ts

```diff
@@ -0,0 +1,10 @@
+import knit = require('@selrond/knit');
```

Comment:
```suggestion
import { knit } from '@selrond/knit';
```
than other changes would work (using named import directly)